terminus-ui/scss/helpers/_shadows.spec.scss

```diff
@@ -1,2 +1,729 @@
+          (0, 1, 8, 0, rgba(0, 0, 0, .12))
+        )
+      );
+
+      @include assert-equal($output, $expect);
+    }
+
+    @include test('should set on elevation of 3') {
+      $output: find-bounding-shadow-sets(3);
+      $expect: (
+        elevation: 3,
+        shadows: (
+          (0, 3, 3, -2, rgba(0, 0, 0, .2)),
+          (0, 3, 4, 0, rgba(0, 0, 0, .14)),
+          (0, 1, 8, 0, rgba(0, 0, 0, .12)))
+        ),
+        (elevation: 4,
+        shadows: (
+          (0, 2, 4, -1, rgba(0, 0, 0, .2)),
+          (0, 4, 5, 0, rgba(0, 0, 0, .14)),
+          (0, 1, 10, 0, rgba(0, 0, 0, .12))
+        )
+      );
+
+      @include assert-equal($output, $expect);
+    }
+
+    @include test('should set on elevation of 4') {
+      $output: find-bounding-shadow-sets(4);
+      $expect: (
+        elevation: 4,
+        shadows: (
+          (0, 2, 4, -1, rgba(0, 0, 0, .2)),
+          (0, 4, 5, 0, rgba(0, 0, 0, .14)),
+          (0, 1, 10, 0, rgba(0, 0, 0, .12)))
+        ),
+        (elevation: 6,
+        shadows: (
+          (0, 3, 5, -1, rgba(0, 0, 0, .2)),
+          (0, 6, 10, 0, rgba(0, 0, 0, .14)),
+          (0, 1, 18, 0, rgba(0, 0, 0, .12))
+        )
+      );
+
+      @include assert-equal($output, $expect);
+    }
+
+    @include test('should set on elevation of 6') {
+      $output: find-bounding-shadow-sets(6);
+      $expect: (
+        elevation: 6,
+        shadows: (
+          (0, 3, 5, -1, rgba(0, 0, 0, .2)),
+          (0, 6, 10, 0, rgba(0, 0, 0, .14)),
+          (0, 1, 18, 0, rgba(0, 0, 0, .12)))
+        ),
+        (elevation: 8,
+        shadows: (
+          (0, 5, 5, -3, rgba(0, 0, 0, .2)),
+          (0, 8, 10, 1, rgba(0, 0, 0, .14)),
+          (0, 3, 14, 2, rgba(0, 0, 0, .12))
+        )
+      );
+
+      @include assert-equal($output, $expect);
+    }
+
+    @include test('should set on elevation of 8') {
+      $output: find-bounding-shadow-sets(8);
+      $expect: (
+        elevation: 8,
+        shadows: (
+          (0, 5, 5, -3, rgba(0, 0, 0, .2)),
+          (0, 8, 10, 1, rgba(0, 0, 0, .14)),
+          (0, 3, 14, 2, rgba(0, 0, 0, .12)))
+        ),
+        (elevation: 16,
+        shadows: (
+          (0, 8, 10, -5, rgba(0, 0, 0, .2)),
+          (0, 16, 24, 2, rgba(0, 0, 0, .14)),
+          (0, 6, 30, 5, rgba(0, 0, 0, .12))
+        )
+      );
+
+      @include assert-equal($output, $expect);
+    }
+
+    @include test('should set on elevation of 16') {
+      $output: find-bounding-shadow-sets(16);
+      $expect: (
+        elevation: 8,
+        shadows: (
+          (0, 5, 5, -3, rgba(0, 0, 0, .2)),
+          (0, 8, 10, 1, rgba(0, 0, 0, .14)),
+          (0, 3, 14, 2, rgba(0, 0, 0, .12)))
+        ),
+        (elevation: 16,
+        shadows: (
+          (0, 8, 10, -5, rgba(0, 0, 0, .2)),
+          (0, 16, 24, 2, rgba(0, 0, 0, .14)),
+          (0, 6, 30, 5, rgba(0, 0, 0, .12))
+        )
+      );
+
+      @include assert-equal($output, $expect);
+    }
+  }
+
+/* 
```

Comment:
We should add a comment here to explain why it's commented out.

Reply:
will do